tools/yaml-templates/build-test-publish.yml

```diff
@@ -118,10 +118,10 @@ steps:
 
   - task: CopyFiles@2
     inputs:
-      Contents: |
-        packages\teams-js\dist\**\?(*.js|*.ts|*.map)
+      sourceFolder: 'packages\teams-js\dist'
+      contents: '**\?(*.js|*.ts|*.map)'
       TargetFolder: '$(Build.ArtifactStagingDirectory)\NPMFeed\dist'
-      flattenFolders: true
+      flattenFolders: false
```

Comment:
I assume this is to for backcompat in case a developer doesn't have ESModules configured

Reply:
Yes, that is one reason, but mostly it is because the rollup-built package keeps all of the files separate, so if we flatten all of the files to one folder level, our published artifact would be a huge mess of every file in teams-js condensed to a single folder level. Keeping the folder structure helps keep the artifact neat.

Reply:
Got it, thanks for the explanation